src/react/branching.js

```diff
@@ -165,6 +166,45 @@ function applyBranchedLogicValue(realm: Realm, value: Value): Value {
         );
       }
     );
+  } else if (value instanceof AbstractValue && value.kind === "||") {
```

Comment:
I don't seem to ever get into that branch when I run the tests. How do you get into it?

Reply:
Probably no test touches it.

Reply:
Can you add some? I couldn't figure out how to get it to follow this code path.
I can't verify some things around it otherwise.

Reply:
Same for branch just below.

Reply:
You're right, we can't hit this code path. `someReactNode || someReactNode` and `someReactNode && someReactNode` are not possible at this point (it would always resolve truthy or falsey because they are known objects).